<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GlimpseVFX Houdini File Manager</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QRadioButton" name="kdrive">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>K:</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="wdrive">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>W:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="yearCbox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="projectList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="filterList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>111</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>311</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="departmentList">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="tagEdit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>310</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="versionButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>350</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Version Up</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="shotRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Shot</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="charRadio">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Char</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="envRadio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Env</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="propRadio">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Prop</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="explorerButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>390</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Explorer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="overwriteButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>390</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Overwrite</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
